regtests/search.test.js

```diff
@@ -0,0 +1,34 @@
+import elastic from '../backend/elastic';
+import Keys from '../common/Keys';
+
+function getFirstClassResult(results) {
+  return results.searchContent[0]["class"];
+}
+
+describe('elastic', () => {
+  it('returns specified class with class code query', async () => {
+    let firstResult = getFirstClassResult(await elastic.search('cs2500', '202010', 0, 1));
+    expect(Keys.getClassHash(firstResult)).toBe('neu.edu/202010/CS/2500');
+  });
+
+  it('returns specified class with name query', async () => {
+    let firstResult = getFirstClassResult(await elastic.search('fundamentals of computer science 2', '202010', 0, 1));
+    expect(Keys.getClassHash(firstResult)).toBe('neu.edu/202010/CS/2510');
+  });
+
+  it('returns a professor if name requested', async () => {
+    let results = await elastic.search('mislove', '202010', 0, 1);
```

Comment:
Not sure if there's an easy way to identify employee documents in the same way as class documents.

Reply:
employees have an `id` that might work. All the employees found through [this file](https://github.com/ryanhugh/searchneu/blob/master/backend/scrapers/employees/employees.js) will have one: 

and for the rest of the employees all the data we have on them is hashed and the hash is used as an id: 
https://github.com/ryanhugh/searchneu/blob/f8d3b4feddc1544091cce23f27a1899734bd0cef/backend/scrapers/employees/matchEmployees.js#L341

You can't search through the search index by id (right?).

If that doesn't work using their emails might work.